eclass-writing/text.xml

```diff
@@ -659,6 +659,122 @@ src_compile() {
 }
 </codesample>
 
+<p>
+<!-- TODO
+Eclasses may inherit other eclasses to make use of their functionality, and,
+historically there have been instances of eclasses calling
+<c>EXPORT_FUNCTIONS</c> and then inheriting another eclass. 
+the time, and 
+-->
+
+The PMS now states that an eclass must not rely on any particular behaviour when
+inheriting another eclass <e>after</e> a call to <c>EXPORT_FUNCTIONS</c>.
+</p>
+
+</body>
+</section>
+
+<section>
+<title>Inherit guards</title>
+<body>
+
+<p>
+It is common practice to surround the main contents of an eclass with an
+"inherit guard". Much like header guards known from C, inherit guards help
+ensure that an eclass can be inherited multiple times and have its functions and
+variables defined only once.
+</p>
+
+<p>
+Although it is not nearly as fatal to omit an inherit guard as opposed to a C
+header guard, it is considered good practice to avoid potential confusing
+behaviour that can arise from an eclass being inherited more than once.
+<!-- FIXME expand with more arguments? -->
+</p>
```

Comment:
An inherit guard is only needed for an eclass that can actually be inherited more than once. Otherwise, omitting it is no problem at all. So I'd avoid the word "fatal" here.

Reply:
I removed the paragraph, and reworked the previous one a little. No more "fatal" :-)